command/sub8_missions/nodes/move_command

```diff
@@ -5,8 +5,17 @@ from twisted.internet import defer, reactor
 import numpy as np
 from mil_misc_tools import text_effects
 from sub8 import sub_singleton
-from geometry_msgs.msg import PoseStamped, PointStamped
+from geometry_msgs.msg import PoseStamped, PointStamped, Pose, Point, Quaternion, Twist
+from gazebo_msgs.msg import LinkStates, ModelState
```

Comment:
It looks like LinkStates, publisher, Float64, BaggerCommands and HandlerBase all get imported but never used. 

In particular in BaggerCommands should be removed since @ironmig's recent changes to online bagger mean it is now unavailable.